recipe/meta.yaml

```
@@ -28,7 +28,7 @@ build:
 requirements:
   build:
     - make  # [unix]
-    - cmake <3.29.1
+    - cmake
```

```suggestion
    - cmake !=3.29.1
```

We might want to exclude it entirely, in case the solver decides it is the best fit given other dependencies.